src/patterns/factory/FurnitureFactory.java

```diff
@@ -0,0 +1,22 @@
+package patterns.factory;
+
+public class FurnitureFactory {
+
+    public Furniture getFurniture(FurnitureTypes type) {
+        Furniture result = null;
+        switch (type) {
+            case CHAIR:
+                result = new ChairFurniture();
```

Comment:
So, how do you use the method `create` of each furniture? it seems to be unused, so my question is why do we need it? Can you refactor the code in a way that in the scope of each case block we do something like this: `result = ChairFurniture.create()`?